site/app/views/forum/ForumThreadView.php

```diff
@@ -264,7 +264,23 @@ function changeName(element, user, visible_username, anon){
 					<option value="{$categories[$i]['category_id']}" style="color: {$categories[$i]['color']}">{$categories[$i]['category_desc']}</option>
 HTML;
 			}
-
+	$cookieSelectedCategories = '';
+	$category_ids_array = array_column($categories, 'category_id');
+	foreach(explode('|', $_COOKIE['forum_categories']) as $selectedId) {
```

Comment:
Need additional `!empty($_COOKIE['forum_categories'])` in case cookie doesn't exists.